packages/shared/src/test_utils/test_common.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import { JSDOM } from 'jsdom';
+import { reset } from '../utils/url/url';
+
+const jsdom = new JSDOM();
```

Comment:
Do we need this line? because we don't have it in the master branch

Reply:
Will check @niloo-fs

Reply:
The changeURL is no longer supported in jsdom. This is the new approach to change url. 🙏